packages/compiler-core/__tests__/transforms/vOn.spec.ts

```diff
@@ -285,6 +285,21 @@ describe('compiler: transform v-on', () => {
         },
       ],
     })
+
+    const { node: node2 } = parseWithVOn(
+      `<div @click="(e: (number | stirng)[]) => foo(e)"/>`,
```

Comment:
Hey there - appreciate the fast response to my issue! 🚀  Just wanted to leave as comment that you might have a typo in 'string' here and also down below in the expect.

Reply:
thx!